packages/zarith-ppx/zarith-ppx.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "zarith-ppx"
+version: "0.1"
```

Comment:
not a blocker at all but those two fields are redundant with the information opam-repository already has